challenges.md

```diff
@@ -0,0 +1,44 @@
+# About the Code Challenges
+We have a couple of code challenges designed to give you the opportunity to showcase your skills. We love to see creativity, originality and simplicity of design. All of our projects involve some creativity, research, interest in data, automation and of course amazing instructions. Please make sure to write the most incredible and informative README.md files you can imagine.
+
+There are tons of amazing open and free data sets to play with. Here are a few for inspiration:
+
+* [Data.gov](https://www.data.gov/open-gov/)
+* [NYC Open Data](https://nycopendata.socrata.com/)
+* [AWS Open Data](https://aws.amazon.com/public-data-sets/)
+* [OpenData Baltimore](https://data.baltimorecity.gov/)
+* [Open Data Sets on GitHub](http://www.kdnuggets.com/2015/04/awesome-public-datasets-github.html)
+* [Firebase Data Sets](https://www.firebase.com/docs/open-data/)
```

Comment:
You knew this was coming....

* [Austin, TX Data](https://data.austintexas.gov/browse)